src/aof.c

```diff
@@ -758,7 +758,7 @@ int openNewIncrAofForAppend(void) {
     if (newfd == -1) {
         serverLog(LL_WARNING, "Can't open the append-only file %s: %s",
             new_aof_name, strerror(errno));
-
+        sdsfree(new_aof_name);
```

Comment:
The sdsfree here is not necessary, `new_aof_name` will be released together in the subsequent `aofManifestFree`.